packages/xo-web/src/common/xo/index.js

```diff
@@ -3303,6 +3304,30 @@ export const deployProxyAppliance = (license, sr, { network, proxy, ...props } =
     ...props,
   })::tap(subscribeProxies.forceRefresh)
 
+export const registerProxy = async () => {
+  const registerProxyInfo = await confirm({
+    body: <RegisterProxyModal />,
+    icon: 'connect',
+    title: _('registerProxy'),
```

Comment:
What do you think about adding the link to the XO proxy guide here? 
like: an information icon with a tooltip